provisioning/packer-virtual-machines/docker-image/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:20.04
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -y install apt-utils \
+    gnupg2 \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release && \ 
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg &&\
+    echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null && \
+    apt-get update && \
+    DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y docker-ce \
+    docker-ce-cli \
+    containerd.io
+
+
+WORKDIR /images
+
+ENV IMAGE_NAME=
+ENV USERNAME=
+ENV PASSWORD=
+
+CMD service docker start && \
+    rm -f Dockerfile && \
+    echo 'FROM scratch' >> Dockerfile && \
+    echo 'ADD output.qcow2 /disk/' >> Dockerfile && \
+    docker login -u "$(echo $USERNAME)" -p "$(echo $PASSWORD)" harbor.crownlabs.polito.it && \
+    docker build -t "$(echo $IMAGE_NAME)" . && \
+    docker push "$(echo $IMAGE_NAME)"
```

Comment:
As we already discussed, this should be replace with [kaniko](https://github.com/GoogleContainerTools/kaniko), which is specifically thought to be executed as a container.

Reply:
Agreed to postpone to a follow-up PR